tensorflow_federated/python/core/utils/BUILD

```diff
@@ -115,6 +115,26 @@ py_test(
     ],
 )
 
+py_library(
+    name = "execution_tracing",
```

Comment:
problem is that you're not using this target as a dependency anywhere (bazel will only include what's needed, e.g. dependencies)

Reply:
:dizzy_face: